app/script/auth/localeConfig.js

```diff
@@ -0,0 +1,33 @@
+export const SUPPORTED_LANGUAGE = [
+  'cs',
+  'da',
+  'de',
+  'el',
+  'es',
+  'et',
+  'fi',
+  'fr',
+  'hr',
+  'hu',
+  'it',
+  'lt',
+  'nl',
+  'pl',
+  'pt',
+  'ro',
+  'ru',
+  'sk',
+  'sl',
+  'tr',
+  'uk',
+];
+export const DEFAULT_LANGUAGE = 'en';
+
+export function getLocale() {
+  return navigator.languages && navigator.languages.length ? navigator.languages[0] : navigator.language;
```

Comment:
In my case:
`navigator.languages` -> `["de-DE", "en-XA", "de", "en-US", "en"]`
`navigator.language` -> `"de"`

Because of this `navigator.languages[0]` returns `de-DE` and `navigator.language` returns just `de`.

We need to make sure that we always get the 2 characters locale code (ISO 639-1).

Reply:
Unfortunately that's something we have to deal with. There is a function below this one which handles just the language part though.

Reply:
👍